packages/react-day-picker/src/contexts/DayPicker/defaultContextValues.ts

```diff
@@ -0,0 +1,61 @@
+import { enUS } from 'date-fns/locale';
+
+import { Caption, CaptionLayout } from 'components/Caption';
+import { CaptionLabel } from 'components/CaptionLabel';
+import { Day } from 'components/Day';
+import { DayContent } from 'components/DayContent';
+import { Dropdown } from 'components/Dropdown';
+import { Footer } from 'components/Footer';
+import { Head } from 'components/Head';
+import { IconDropdown } from 'components/IconDropdown';
+import { IconLeft } from 'components/IconLeft';
+import { IconRight } from 'components/IconRight';
+import { Row } from 'components/Row';
+import { WeekNumber } from 'components/WeekNumber';
+import { DayPickerContextValue } from 'contexts/DayPicker';
+
+import { defaultClassNames } from './defaultClassNames';
+import * as formatters from './formatters';
+import * as labels from './labels';
+
+export function getDefaultContextValues(): DayPickerContextValue {
```

Comment:
The test for the `ModifierContext` was easier to mock by putting the default values set by the DayPicker context into a separate function. This also allowed me to cleanup the DayPicker context.